src/app/view/variant/variant-view/variant-history/version-entry-layout/version-entry-layout.component.scss

```diff
@@ -70,3 +70,22 @@
     border: none;
   }
 }
+
+
+.user-id {
+  width: 140px;             /* Set the desired width */
```

Comment:
Are all of these comments really necessary? To me, the code itself is sufficiently self-explanatory…